modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -112,6 +115,8 @@ public DLAdminManagementToolbarDisplayContext(
 		_liferayPortletResponse = liferayPortletResponse;
 		_siteConnectedGroupGroupProvider = siteConnectedGroupGroupProvider;
 
+		_dlFileEntryTypeService = dLFileEntryTypeService;
```

Comment:
Hi @bakayattila , this might be in the wrong place, should nt be in the queue above sorted?

Reply:
done